changelog.md

```diff
@@ -1,3 +1,8 @@
+Unreleased
+----------
+
+* Use html `buttons` instead of `inputs` for form submission.
```

Comment:
It occurs to me that this might be a `v2.0.0` on release, since it might be backwards-incompatible for some CSS styling setups.